website/source/docs/upgrade-specific.html.md

```diff
@@ -48,18 +48,22 @@ in OSS or `web.service.ns1.dc1.consul` for Enterprise.
 
 Consul has changed the semantics of query counts in its [telemetry](/docs/agent/telemetry.html#metrics-reference).
 `consul.rpc.query` now only increments on the *start* of a query (blocking or non-blocking), whereas before it would
-measure when blocking queries polled for more data. The gauge `consul.rpc.queries_blocking` has been added for a more
+measure when blocking queries polled for more data. The `consul.rpc.queries_blocking` gauge has been added
 to more precisely capture the view of *active* blocking queries.
 
-### Vault: default for `http_max_conns_per_client` to low to run Vault properly
+### Vault: default `http_max_conns_per_client` too low to run Vault properly
 
-Consul 1.7.0 introduced limiting of connections per client ((docs)[https://www.consul.io/docs/agent/options.html#http_max_conns_per_client]). The default is 100, but Vault allows up to 128 which causes problems. If you want to use Vault with Consul 1.7.0, you should change it to 200. Starting with Consul 1.7.1 this is the new default.
+Consul 1.7.0 introduced [limiting of connections per client](/docs/agent/options.html#http_max_conns_per_client). The default value
+was 100, but Vault used up to 128 which caused problems. If you want to use Vault with Consul 1.7.0, you should change the value to 200.
+Starting with Consul 1.7.1 this is the new default.
 
 ## Consul 1.6.3
 
-### Vault: default for `http_max_conns_per_client` to low to run Vault properly
+### Vault: default `http_max_conns_per_client` too low to run Vault properly
 
-Consul 1.6.3 introduced limiting of connections per client ((docs)[https://www.consul.io/docs/agent/options.html#http_max_conns_per_client]). The default is 100, but Vault allows up to 128 which causes problems. If you want to use Vault with Consul 1.6.3, you should change it to 200. Starting with Consul 1.6.4 this is the new default.
+Consul 1.6.2 introduced [limiting of connections per client](/docs/agent/options.html#http_max_conns_per_client). The default value
```

Comment:
Previously this said `1.6.3` both introduced the problem and fixed the problem. I'm fairly sure that is wrong. I guessed that 1.6.2 introduced the problem, but I haven't confirmed this is true yet. It may have been an earlier version. I'll see if I can confirm.

Reply:
I am reading the original as saying that 1.6.3 introduced the problem, you can mitigate with setting a configuration item and that the default was updated in 1.6.4. The original seems accurate.

Connection limiting was introduced in 1.6.3 which you can see in the changelog here: https://github.com/hashicorp/consul/blob/master/CHANGELOG.md#163-january-30-2020

Reply:
Ah, the heading is `## Consul 1.6.3`. So the error is the heading should say `## Consul 1.6.4`. I will fix.

Reply:
I think I was reading this wrong. The header is not the fix version, but the problematic version. So the versions were correct, I was confused.